jenkins/scripts/integration_clean.sh

```diff
@@ -22,10 +22,17 @@ function cleanup() {
     startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-vm-") |
     split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 21600))
     | .ID ')
+
+  DEBUGGING_VM_LIST=$(openstack server list -f json | jq -r '.[] | select(.Name |
+  startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-keep-vm-") |
+  split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 86400))
```

Comment:
Could we replace the magic numbers 21600 and 86400 with variables that indicate the length of time in a human readable way?

Reply:
I've added a comment on that